js/quiz.js

```diff
@@ -404,6 +404,10 @@
 
   $(".quiz.button, #quiz .finished .again, .new-popup .quiz").on('click', function () {
 
+    gtag("event", "quiz_onClick", {
```

Comment:
השם לא מדוייק - אתה לא בודק לחיצות על החידון, אתה בודק התחלה של חידון
אני מציע
```suggestion
    gtag("event", "quiz_onStart", {
```

Reply:
personally , doesnt see big difference in the naming but i'll do as you wish.

Reply:
לא רלוונטי

Reply:
כל הכבוד